images/macos/tests/RubyGem.Tests.ps1

```diff
@@ -0,0 +1,39 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
```

Comment:
could you please double check that we really need this import?
It is already imported in https://github.com/actions/virtual-environments/blob/main/images/macos/helpers/Tests.Helpers.psm1#L1 and should be required to import again?

Reply:
Thanks.Done.